website/docs/zh/api/javascript-api/swc.mdx

```diff
@@ -0,0 +1,119 @@
+# SWC API
+
+Rspack 支持 JavaScript 的压缩和转换。Rspack 底层使用 SWC 来转换和压缩 JavaScript 代码，你可以使用 `rspack.experiments.swc` API 来访问这些函数。
+
+## 示例
+
+以下是一个简单的示例，演示如何使用 Rspack 转换 JavaScript 代码：
+
+```js
+import { rspack } from '@rspack/core';
+
+const swc = rspack.experiments.swc;
+
+swc
+  .transform('const a:number = 10;', {
+    filename: 'main.ts',
+    jsc: {
+      parser: {
+        syntax: 'typescript',
+      },
+      target: 'es5',
+    },
+  })
+  .then(result => {
+    console.log(result.code);
+  });
+```
+
+除了直接使用 `swc` API，你也可以在编写 loader 或 plugin 中使用它,来帮助你的进行代码转换或压缩。
+
+```js
+import { rspack } from '@rspack/core';
```

Comment:
In practice, we do not recommend that loaders or plugins install @rspack/core dependencies separately, as this may cause multiple instances.

For plugins, we can recommend using `compiler.rspack.*`, but for loaders, we may need to expose the SWC API on the loader Context, similar to [loaderContext.utils](https://rspack.dev/api/loader-api/context#thisutils) (this can be done in a separate PR)

Reply:
Done~, I add `swc` api to loaderContext.utils also